js/shared/src/logistics/redux/selectors.js

```diff
@@ -84,6 +84,17 @@ export const selectAllTours = createSelector(
   (allTours) => allTours
 )
 
+export const selectTourPolylines = createSelector(
+  selectAllTours,
+  (allTours) => {
+    let polylines = {}
+    allTours.forEach(tour => {
```

Comment:
nit picking but the use of `reduce()` would be cleaner here

Reply:
I use an object as accumulator and i set the key when iterating?